.travis.yml

```diff
@@ -6,12 +6,16 @@ compiler:
 
 matrix:
   include:
-    - env: NIGHTLY_TEST_SETTINGS=true QTHREAD_AFFINITY=no HWLOC_HIDE_ERRORS=1 CHPL_SMOKE_SKIP_DOC=true TEST_COMMAND="./util/buildRelease/smokeTest"
-    - env: NIGHTLY_TEST_SETTINGS=true CHPL_SMOKE_SKIP_MAKE_CHECK=true TEST_COMMAND="./util/buildRelease/smokeTest"
-    - env: TEST_COMMAND="make test-venv && CHPL_HOME=$PWD ./util/run-in-venv.bash ./util/test/check_annotations.py"
+    - name: "make check"
+      env: NIGHTLY_TEST_SETTINGS=true QTHREAD_AFFINITY=no HWLOC_HIDE_ERRORS=1 CHPL_SMOKE_SKIP_DOC=true TEST_COMMAND="./util/buildRelease/smokeTest"
+    - name: "make check-chpldoc"
```

Comment:
I might label this as "includes chpldoc" or the other one as "skips chpldoc".  This label makes it seem like there's a separate script specifically to check that chpldoc builds

Reply:
This one runs the command `make check-chpldoc`, and the other runs the command `make check`

Reply:
If I'm reading smokeTest correctly, the second one runs *both* `make check` and `make check-chpldoc`

Reply:
We set `CHPL_SMOKE_SKIP_MAKE_CHECK` for the second run. It should only be running `make check-chpldoc` (and it looks like `make docs` as well)

Reply:
Oh dur, I was reading the wrong line

Reply:
:). So switch the label to `make check-chpldoc && make docs` and call it good?

Reply:
Works for me!